cli/multisig.go

```diff
@@ -521,7 +525,7 @@ var msigApproveCmd = &cli.Command{
 		},
 	},
 	Action: func(cctx *cli.Context) error {
-		if cctx.NArg() < 2 {
+		if cctx.NArg() != 2 {
```

Comment:
ditto, but there's 2 more blocks below here; so these commands already strictly check their number of arguments, they just have nuance to the way they report the misuse, let's leave it alone